includes/class-wc-subscriptions-manager.php

```diff
@@ -145,8 +145,8 @@ public static function process_renewal( $subscription_id, $required_status, $ord
 				$renewal_order = wcs_create_renewal_order( $subscription );
 
 				if ( is_wp_error( $renewal_order ) ) {
-					// translators: placeholder is an order note.
-					throw new Exception( sprintf( __( 'Error: Unable to create renewal order with note "%s"', 'woocommerce-subscriptions' ), $order_note ) );
+					// translators: placeholder %1 is an order note. %2 is the error message that was thrown.
+					throw new Exception( sprintf( __( 'Error: Unable to create renewal order with note "%1$s". Message: %2$s', 'woocommerce-subscriptions' ), $order_note, $renewal_order->get_error_message() ) );
```

Comment:
I just added this change so the scheduled action will actually capture what the error was, rather than just log a generic "it failed" kind of error. 

<img width="313" alt="Screenshot 2023-12-21 at 3 33 57 pm" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/8490476/e09fad51-7d63-441f-81af-87745c664ca5">